docs/modeling/index.rst

```diff
@@ -10,337 +10,121 @@ Introduction
 ============
 
 `astropy.modeling` provides a framework for representing models and performing
-model evaluation and fitting. It currently supports 1-D and 2-D models and
-fitting with parameter constraints.
+model evaluation and fitting. The goal of this package is to eventually provide
+a rich toolset of models and fitters such that most users will not need to
+define new model classes, nor special purpose fitting routines (while making it
+reasonably easy to do when necessary). See a
+:ref:`quick tutorial <modeling_tutorial>` of the package.
 
-It is designed to be easily extensible and flexible.  Models do not reference
-fitting algorithms explicitly and new fitting algorithms may be added without
-changing the existing models (though not all models can be used with all
-fitting algorithms due to constraints such as model linearity).
+The major components of the package are:
 
-The goal is to eventually provide a rich toolset of models and fitters such
-that most users will not need to define new model classes, nor special purpose
-fitting routines (while making it reasonably easy to do when necessary).
+Models
+------
 
-.. note::
+Models behave like parametrized functions that can be easily evaluated on
+datasets. For example, the `~astropy.modeling.functional_models.Gaussian1D`
+model evaluates the Gaussian function
+:math:`f(x) = A e^{- \frac{\left(x - x_{0}\right)^{2}}{2 \sigma^{2}}}`.
+Single values or arrays can be passed to models to evaluate the function at
+these points. Examples on using models can be found
+:ref:`here <modeling_tutorial>`.
```

Comment:
Instead of "here" maybe write out "the modeling tutorial" just so users know where they're being taken.